Le Soir.js

```diff
@@ -0,0 +1,262 @@
+{
+	"translatorID": "79155be5-1836-4fcb-a4a0-a2e216befa3a",
+	"label": "Le Soir",
+	"creator": "Alexandre Bossard",
+	"target": "^https?://(www\\.)?(plus\\.)?lesoir\\.be/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-11-28 11:52:36"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 Alexandre Bossard
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes("/article/")) {
+		return "newspaperArticle";
+	}
+	else if (url.includes("/recherche?")) {
+		return "multiple";
+	}
+	return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//li[@class="article-inline small gr-article-payant"]/a');
+	for (var i = 0; i < rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (items) ZU.processDocuments(Object.keys(items), scrape);
+		});
+	}
+	else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	// translator.setDocument(doc);
+	translator.setHandler('itemDone', function (obj, item) {
+		item.date = ZU.strToISO(ZU.xpathText(doc, '//time[@class=" pubdate updated"]'));
+		item.publicationTitle = "Le Soir";
+		item.ISSN = "1375-5668";
+		var authorString = ZU.xpathText(doc, '//div[@class="gr-article-infos"]/p[@class="gr-meta gr-meta-author"]|//div[@class="required-fields gr-infos-author"]/p[@class="gr-meta gr-meta-author"]');
+		if (authorString) {
+			authorString = authorString.replace('Par ', '');
```

Comment:
Should use a regex to make sure someone with `Par ` in their name doesn't give us problems :)

```js
authorString = authorString.replace(/^\s*Par\s*/, '');
```